src/%IPM/Repo/Filesystem/Definition.cls

```diff
@@ -39,6 +39,7 @@ repo -name LocalFiles -snapshots 1 -fs -depth 2 -path C:\MyWorkspace\RootModuleD
 </commands>
 }
 
+
```

Comment:
?

Reply:
Hey Dmitry, good catch! Have removed the unnecessary empty line.